docs/migration-guides/v4/color-palette.md

```diff
@@ -1,6 +1,6 @@
 # Deprecated Color Variables
 
-In V4, we are using Spark tokens for colors and will no longer be exporting the following color palette variables. Below is a list of color variables and their hex equivalents.
+In V4, we are using Spark tokens for colors and will no longer be exporting the following color palette variables. Below is a list of color variables and their hex equivalents. Read more about the new Spark 4.0 Design System Color palette on the [Spark Design System documentation site](https://spark.genesys.com/latest/foundations/color/core-palette/design-XYztDeyk)
```

Comment:
Perhaps: 

`Design System Color palette` -> `Design System color palette `

Reply:
I would probably vote for `Design System color palette`